docs/configuration/ping.md

```diff
@@ -27,7 +27,7 @@ The `/ping` health-check URL is enabled with the command-line `--ping` or config
 Thus, if you have a regular path for `/foo` and an entrypoint on `:80`, you would access them as follows:
 
 * Regular path: `http://hostname:80/foo`
-* Admin panel: `http://hostname:8080/`
```

Comment:
Could you remove this change from the PR, since it is out the scope.

Reply:
Thank you for the review. Commit has been reverted.